data-analytics/cloud-composer-examples/composer_http_post_example/make_iap_request.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Example use of a service account to authenticate to Identity-Aware Proxy."""
+
+# [START iap_make_request]
```

Comment:
This style is different because it was pulled directly from [GoogleCloudPlatform/python-docs-samples/master/iap/make_iap_request.py](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/iap/make_iap_request.py) as mentioned in the README. The thought here is that we don't want to be responsible for maintaining this code but in case the user forgets they have a copy in this repo. Open to suggestions of a more elegant way of doing this or communicating this.

Reply:
If it's in the repo, then we're maintaining it. The way things are right now, we won't get any updates that are made to the file from the parent repo. We should just drop this entirely, and have the user retrieve this from the source repo.

Reply:
Deleted